osmnx/simplification.py

```diff
@@ -9,10 +9,11 @@
 import geopandas as gpd
 import networkx as nx
 import numpy as np
-from shapely import LineString
-from shapely import MultiPolygon
-from shapely import Point
-from shapely import Polygon
+import pandas as pd
+from shapely.geometry import LineString
```

Comment:
These geometry imports can be left as `from shapely import ...`

Reply:
I haven't touched those lines in this PR, I would like to keep this changeset atomic. Also maybe good to leave it explicit over implicit.